kenji_crosland/test/test.js

```diff
@@ -10,5 +10,5 @@ describe('the greet function', function(){
 
   it('should process the arguments input in the console', function(){
     expect(greeting.greet(process.argv)).to.eql('hello ' + process.argv);
```

Comment:
Do you really want to pass `process.argv` here?  Or is there a specific piece of `process.argv` that you could use instead?  Think about these: `[ ]`